tests/contentmethods/data/expected.json

```diff
@@ -0,0 +1,16 @@
+{
+    "output_get_content_information": {
```

Comment:
what does this name mean? that as cryptic as it gets. It should be descriptive... what does this mapping contain?